src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -153,9 +150,9 @@ public HttpConnection(Uri url, ITransportFactory transportFactory, ILoggerFactor
             _scopeDisposable = _logger.BeginScope(_logScope);
         }
 
-        public async Task StartAsync() => await StartAsyncCore().ForceAsync();
+        public async Task StartAsync(TransferFormat transferFormat) => await StartAsyncCore(transferFormat).ForceAsync();
```

Comment:
Make the default Text?

Reply:
Not sure I agree. A "socket" is a binary concept by default. It's only because SignalR uses a text-based protocol (JSON) that we want to be able to set the mode to Text (for WebSocket debuggability). I'd rather the default for Endpoints be `Binary` to keep aligned with standard Sockets.

Reply:
I misread a little where this comment was posted (thought you meant the default in DefaultConnectionContext). I'm OK with adding a default (via an overload because I think default arguments are The Worst), I'd still think it should be Binary.